src/actions/guides/frontend/internationalization.cr

```diff
@@ -0,0 +1,341 @@
+class Guides::Frontend::Internationalization < GuideAction
+  guide_route "/frontend/internationalization"
+
+  def self.title
+    "Internationalization - Handle Multiple Languages"
+  end
+
+  def markdown : String
+    <<-MD
+    This describes how to handle USERS with multiple language preferences.  At somepoint we can document how to add dynamic switching from the frontend too.
+
+    ## **Step 1** - add i18n shard to `shard.yml`
```

Comment:
The ** is not rendered well in the table of contents. I think the bold can be removed. Also could you capitalize the sentences after the `-`

Reply:
fixed